FantasmoSDK/app/src/main/java/com/fantasmo/sdk/models/FMFrame.kt

```diff
@@ -77,11 +77,12 @@ class FMFrame (private val frame: Frame,
     }
 
     fun imageData(): ByteArray? {
-        val imageBitmap = yuvImage?.let { yuvToRgbConverter?.toBitmap(it) }
-        imageBitmap?.rotate(getImageRotationDegrees(context))
-        val data = imageBitmap?.let { getFileDataFromDrawable(it) }
+        val image = yuvImage ?: return null
```

Comment:
This is maybe nit picky but for null error handling, is there a reason to combine that with the Elvis operator assignment like this? This seems more idiomatic: 

```
if (yuvImage == null) return null
val image = yuvImage
```

I just tend to associate that kind of operator with an assignment operation but I could just be out of touch here